pkg/search/postgres/common.go

```diff
@@ -581,6 +582,7 @@ func RunSearchRequest(ctx context.Context, category v1.SearchCategory, q *v1.Que
 	schema := mapping.GetTableFromCategory(category)
 
 	return pgutils.Retry2(func() ([]searchPkg.Result, error) {
+		ctx := contextutil.WithValuesFrom(context.Background(), ctx)
```

Comment:
I don't understand this :(
Why we need to use `context.Background` here? How do we propagate timeouts or cancelation then?